extra/modules/live-intent-omni-channel-identity/src/main/java/org/prebid/server/hooks/modules/liveintent/omni/channel/identity/v1/hooks/LiveIntentOmniChannelIdentityProcessedAuctionRequestHook.java

```diff
@@ -40,6 +42,8 @@ public class LiveIntentOmniChannelIdentityProcessedAuctionRequestHook implements
     private final JacksonMapper mapper;
     private final HttpClient httpClient;
     private final double logSamplingRate;
+    private final ActivityImpl enriched = ActivityImpl.of("liveintent-enriched", "success", List.of());
+    private final ActivityImpl treatmentRate;
```

Comment:
Remove activities from object properties. Create them when needed (even if it's always needed).